examples/sdp3x_default_address/sdp3x_default_address.ino

```diff
@@ -9,17 +9,18 @@ void setup() {
   Wire.begin();
   Serial.begin(9600);
   delay(1000); // let serial console settle
-
-  int ret = sdp.init();
-  if (ret == 0) {
+  bool error = false;
```

Comment:
The only purpose of `error` seems to be to keep the loop going. In this case it might make more sense to have an infinite loop which exits (`break;`) on success - that doesn't require an additional variable.